three.c

```diff
@@ -0,0 +1,58 @@
+/* Jared Wasserman -- three.c */
```

Comment:
Great job! Personally would have handled floats differently (not truncated them), but nevertheless, awesome job!